packages/labs/ssr-react/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@lit-labs/ssr-react",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Lit SSR integration for React",
+  "license": "BSD-3-Clause",
+  "author": "Google LLC",
+  "homepage": "https://github.com/lit/lit/tree/main/packages/labs/ssr-react",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/labs/ssr-react"
+  },
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "node": "./node/index.js",
+      "default": "./index.js"
+    },
+    "./enable-lit-ssr.js": {
+      "types": "./enable-lit-ssr.d.ts",
+      "node": "./node/enable-lit-ssr.js",
+      "default": "./enable-lit-ssr.js"
+    },
+    "./jsx-dev-runtime": {
+      "types": "./jsx-dev-runtime.d.ts",
+      "node": "./node/jsx-dev-runtime.js",
+      "default": "./jsx-dev-runtime.js"
+    },
+    "./jsx-runtime": {
+      "types": "./jsx-runtime.d.ts",
+      "node": "./node/jsx-runtime.js",
+      "default": "./jsx-runtime.js"
+    }
+  },
+  "files": [
+    "/lib/",
+    "/node/",
+    "/enable-lit-ssr.{d.ts,d.ts.map,js,js.map}",
+    "/index.{d.ts,d.ts.map,js,js.map}",
+    "/jsx-dev-runtime.{d.ts,d.ts.map,js,js.map}",
+    "/jsx-runtime.{d.ts,d.ts.map,js,js.map}"
+  ],
+  "dependencies": {
+    "@lit-labs/ssr": "^3.0.1",
+    "lit": "^2.6.1"
+  },
+  "peerDependencies": {
+    "@types/react": "17 || 18",
+    "react": "17 || 18"
+  },
+  "scripts": {
+    "build": "wireit",
+    "build:ts": "wireit",
+    "test": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "../ssr:build:ts"
+      ],
+      "files": [
+        "src/**/*.ts{,x}",
+        "tsconfig.json"
+      ],
+      "output": [
+        "lib/",
+        "node/",
+        "test/",
+        "enable-lit-ssr.{d.ts,d.ts.map,js,js.map}",
+        "index.{d.ts,d.ts.map,js,js.map}",
+        "jsx-dev-runtime.{d.ts,d.ts.map,js,js.map}",
+        "jsx-runtime.{d.ts,d.ts.map,js,js.map}",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "test": {
+      "command": "uvu test _test.js$",
+      "files": [],
+      "output": [],
+      "dependencies": [
+        "build"
+      ]
+    }
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.27",
+    "@types/react-dom": "^18.0.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "uvu": "^0.5.6"
+  },
+  "keywords": [
+    "lit",
+    "lit-html",
+    "lit-element",
+    "LitElement",
+    "ssr",
+    "lit-ssr",
+    "react"
```

Comment:
add "web components" and "next". If there's a limit, I don't know if we need `lit-element` and variations.

Reply:
Done